.phpstan/phpstan-baseline.neon

```diff
@@ -0,0 +1,206 @@
+
+		-
+		    # will be fixed in v4. Currently BC break
+			message: "#^Method Sonata\\\\AdminBundle\\\\Admin\\\\FieldDescriptionInterface\\:\\:getLabel\\(\\) invoked with 1 parameter, 0 required\\.$#"
+			count: 1
+			path: ../src/Show/ShowMapper.php
+
+		-
+		    # will be fixed in v4. Currently BC break
+			message: "#^Method Sonata\\\\AdminBundle\\\\Admin\\\\AdminInterface\\:\\:getShowGroups\\(\\) invoked with 1 parameter, 0 required\\.$#"
+			count: 1
+			path: ../src/Show/ShowMapper.php
+
+		-
+		    # will be fixed in v4. Currently BC break
+			message: "#^Method Sonata\\\\AdminBundle\\\\Admin\\\\AdminInterface\\:\\:getShowTabs\\(\\) invoked with 1 parameter, 0 required\\.$#"
+			count: 1
+			path: ../src/Show/ShowMapper.php
+
+		-
+		    # will be fixed in v4. Currently BC break
+			message: "#^Method Sonata\\\\AdminBundle\\\\Translator\\\\Extractor\\\\JMSTranslatorBundle\\\\AdminExtractor\\:\\:buildSecurityInformation\\(\\) should return array\\<string, mixed\\> but return statement is missing\\.$#"
+			count: 1
+			path: ../src/Translator/Extractor/JMSTranslatorBundle/AdminExtractor.php
+
+		-
+		    # will be fixed in v4. Currently BC break
+			message: "#^Method Sonata\\\\AdminBundle\\\\Translator\\\\Extractor\\\\JMSTranslatorBundle\\\\AdminExtractor\\:\\:getBaseRole\\(\\) should return string but return statement is missing\\.$#"
+			count: 1
+			path: ../src/Translator/Extractor/JMSTranslatorBundle/AdminExtractor.php
+
+# next 6 errors are due to not installed Doctrine ORM\ODM
+		-
+			message: "#^Class Doctrine\\\\ODM\\\\MongoDB\\\\PersistentCollection not found\\.$#"
+			count: 1
+			path: ../src/Admin/AdminHelper.php
+
+		-
+			message: "#^Class Doctrine\\\\ORM\\\\PersistentCollection not found\\.$#"
+			count: 1
+			path: ../src/Admin/AdminHelper.php
+
+		-
+			message: "#^Call to method getTypeClass\\(\\) on an unknown class Doctrine\\\\ODM\\\\MongoDB\\\\PersistentCollection\\.$#"
+			count: 1
+			path: ../src/Admin/AdminHelper.php
+
+		-
+			message: "#^Call to method getTypeClass\\(\\) on an unknown class Doctrine\\\\ORM\\\\PersistentCollection\\.$#"
+			count: 1
+			path: ../src/Admin/AdminHelper.php
+
+		-
+			message: "#^Call to method add\\(\\) on an unknown class Doctrine\\\\ODM\\\\MongoDB\\\\PersistentCollection\\.$#"
+			count: 1
+			path: ../src/Admin/AdminHelper.php
+
+		-
+			message: "#^Call to method add\\(\\) on an unknown class Doctrine\\\\ORM\\\\PersistentCollection\\.$#"
+			count: 1
+			path: ../src/Admin/AdminHelper.php
+
+# Symfony related errors
+		-
+			message: "#^Call to an undefined method Symfony\\\\Component\\\\Config\\\\Definition\\\\Builder\\\\NodeParentInterface\\:\\:arrayNode\\(\\)\\.$#"
+			count: 1
+			path: ../src/DependencyInjection/Configuration.php
+
+		-
+		    # Symfony BC break policy
+			message: "#^Method Symfony\\\\Contracts\\\\EventDispatcher\\\\EventDispatcherInterface\\:\\:dispatch\\(\\) invoked with 2 parameters, 1 required\\.$#"
+			count: 1
+			path: ../src/Menu/MenuBuilder.php
```

Comment:
Why use tabs here? Is that a standard? It would be the only file in the project that does so, and besides, the `phpstan.neon.dist` below uses spaces.

Reply:
I think I'm done with this :crossed_fingers: 

as I said before, I thought that PhpStorm will handle this